src/frontends/tensorflow/src/helper_transforms/block_lstm_replacer.hpp

```diff
@@ -18,7 +18,7 @@ namespace pass {
 
 // This transformation replaces BlockLSTM with such outputs as concatenated hidden states
 // and cell state from the last time step.
-class TENSORFLOW_API BlockLSTMReplacer : public ov::pass::MatcherPass {
```

Comment:
Do we have any other places where we have exported internal functions from the frontend?

Reply:
Only in `include` directory for public TF FE API. The `src` directory does not contain it.